### ISSUE TITLE
The output is almost bad.

### ISSUE THREAD

**Issue by milanyummy** [D-X-Y/caffe-reid#5]
Hello！
I followed your instructions and trained a res50 model. But when I used examples/market1501/extract/extract_prediction.sh to try to extract the features. The output was almost bad. Is this normal?

**Comment by D-X-Y** [comment]
How about your loss values in the training stage?

**Comment by D-X-Y** [comment]
I update the readme, the model is just trained one day ago, using the right loss weights.
Some detailed parameters such lr_multi may be different with the official paper.

**Comment by milanyummy** [comment]
OK, I will try the new one.Thanks for your reply.

**Comment by milanyummy** [comment]
Hello!
I tried the new one and the final loss value after 18000 lterations was 0.0464386 (Iteration 18000, loss = 0.0464386 [ 18000 / 18000 ] -> [ 0 s / 0:0 (H:M) ])
Then I used  `sh examples/market1501/extract/extract_prediction.sh 0 res50 pool5` and the final output was  "3368 /  3368 images, need 00:00:00. [PRED: 402] vs [OK: 1500] accuracy: 0.0009 = good:     3 bad:  3365".

**Comment by D-X-Y** [comment]
This means nothing, just for extracting feature. See readme for the full evaluation method,
